File: MORM.Java/src/MORM/Java/Models/Venda/Cfop.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "CFOP")
public class Cfop extends CollectionItem
{
    @CampoAttribute(Campo = "CD_CFOP", Tipo = CampoTipo.Key)
    public Integer Cd_Cfop;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "DS_CFOP", Tipo = CampoTipo.Req)
    public String Ds_Cfop;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Cfop.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("CFOP")]
    public class Cfop : CollectionItem
    {
        [Campo("CD_CFOP", CampoTipo.Key)]
        public int Cd_Cfop { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("DS_CFOP", CampoTipo.Req)]
        public string Ds_Cfop { get; set; }
    }
}